File: src/main/java/Pages/ModalConfirmPage.java
package Pages;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;



public class ModalConfirmPage extends BasePage {

    By msjH2 = By.xpath("//div[@class='sweet-alert  showSweetAlert visible']/child::h2");
    By btnConfirm = By.xpath("//button[@class='confirm btn btn-lg btn-primary']");
    By txtConfirm = By.xpath("//p[@class='lead text-muted ']");




    WebDriver driver;

    public ModalConfirmPage(WebDriver driver) {
        this.driver = driver;

    }

    public String GetMsjConfirm(){
        return driver.findElement(msjH2).getText();

    }

    public void ClickConfirmOk(){
        driver.findElement(btnConfirm).click();
    }

    public String getTxtConfirm(){
        return driver.findElement(txtConfirm).getText();

    }




}

File: test/pageobjects/modalConfirmPage.js
const Page = require('./page');


class modalConfirmPage extends Page {
  

    get titleConfirm () {
        return $("//div[@class='sweet-alert  showSweetAlert visible']/child::h2");
    }

    
    get textConfirm () {
        return $("//p[@class='lead text-muted ']");
    }

    
    get btnConfirm () {
        return $("//button[@class='confirm btn btn-lg btn-primary']");
      
    }


    clicktoConfirmOK(){
       super.clicktoBtn(this.btnConfirm);
    }

    getTitleConfirm(){
        return this.titleConfirm.getText()
    }

    getTextConfirm(){
        return this.textConfirm.getText()
    }

   
  
}

module.exports = new modalConfirmPage();
